fe/fe-core/src/main/java/com/starrocks/monitor/unit/ByteSizeValue.java

```diff
@@ -92,22 +92,22 @@ public double getPbFrac() {
     public String toString() {
         long bytes = getBytes();
         double value = bytes;
-        String suffix = "b";
+        String suffix = "B";
```

Comment:
If this is changed, should the units in parseBytesSizeValue and simpleParseBytesSizeValue also be changed? But then again, these two methods don't seem to work, should that be cleaned up?